requirements.txt

```diff
@@ -2,11 +2,11 @@
 flask
 gunicorn
 python-dotenv
-authlib
 requests
 flask-sqlalchemy
 psycopg2-binary
 tqdm
 
 git+https://github.com/CCRI-POPROX/poprox-concepts.git
 git+https://github.com/CCRI-POPROX/poprox-storage.git
+git+https://github.com/CCRI-POPROX/poprox-platform.git
```

Comment:
Thats about where Michael and I got to "probably a bad sign, but good for now"

Reply:
FWIW I'm actively testing this code locally -- so there are not any snags (yet) but I also haven't gotten the emails I've been sending myself because they have a null unsubscribe link (and we haven't landed the full fix to that)